Emby.Server.Implementations/Plugins/PlugInManifest.cs

```diff
@@ -0,0 +1,32 @@
+#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
```

Comment:
Please document this file, even if it's just 

```
/// <summary>
/// Gets or sets the ${propertyName}.
/// </summary>


Also, formatting for this file is off

Reply:
Sorry forgot!